framework/packages/abstract-std/src/native/registry.rs

```diff
@@ -58,27 +58,8 @@ pub mod state {
     /// Account sequences
     pub const LOCAL_ACCOUNT_SEQUENCE: Item<AccountSequence> =
         Item::new(storage_namespaces::registry::LOCAL_ACCOUNT_SEQUENCE);
-    /// Sub indexes for namespaces.
```

Comment:
This is needed, because we use idx at some point, so what's the decision here ?

Reply:
What do you mean? One account can only have one namespace. And indexedmap is overkill just for that

Reply:
Ok, reverse namespace it is